OWNERS

```diff
@@ -7,9 +7,13 @@ approvers:
   - cluster-api-maintainers
 
 emeritus_approvers:
-  - roberthbailey
-  - kris-nova
   - chuckha
+  - kris-nova
+  - ncdc
+  - roberthbailey
+
+emeritus_maintainers:
```

Comment:
I saw CAPA starting having this section. Not sure if we want both or just keep with approvers.